tests/rkt_exec_test.go

```diff
@@ -41,32 +41,32 @@ func TestRunOverrideExec(t *testing.T) {
 	defer ctx.Cleanup()
 
 	for _, tt := range []struct {
-		rktCmd       string
-		expectedLine string
+		rktCmd        string
+		expectedRegex string
```

Comment:
Since systemd unit files only accept an absolute path in [ExecStart=](https://www.freedesktop.org/software/systemd/man/systemd.service.html#ExecStart=), stage1/init builds that absolute path.

In the test image below, `/bin/inspect-link-bin` is a symlink to `/inspect`. The test executes the command "inspect-link-bin". Previously, appexec would use execvp() directly and the process would have argv[0] == "inspect-link-bin".

But now, the systemd unit has `ExecStart=/bin/inspect-link-bin`, so the process has argv[0] == "/bin/inspect-link-bin".

This change makes the test more flexible: it accepts any path in argv[0] as long as the basename is ok. In practice, it's important to get the correct basename for things like busybox (i.e. we should not resolve the symlinks).